src/plugins/mer/merdeploysteps.cpp

```diff
@@ -93,6 +94,8 @@ namespace {
 const int CONNECTION_TEST_CHECK_FOR_CANCEL_INTERVAL = 2000;
 
 const char MER_RPM_VALIDATION_STEP_SELECTED_SUITES[] = "MerRpmValidationStep.SelectedSuites";
+const char MER_RPM_BUILD_STEP_SIGN[] = "MerRpmBuildStep.SignEnable";
```

Comment:
name and value would be better matching

```suggestion
const char MER_RPM_BUILD_STEP_SIGN_PACKAGES[] = "MerRpmBuildStep.SignPackages";
```